tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -172,6 +179,135 @@ func (rb *IptablesRuleBuilder) buildRules(rules []*Rule) [][]string {
 	return output
 }
 
+func reverseRules(rules []*Rule) []*Rule {
```

Comment:
can we add comments to the function? Given a function signature `reverseRules(rules []*Rule) []*Rule` without comments, I would assume it reverse the order of the rules -- which is not what it does at all

Reply:
Added comments to both reverseRules() and checkRules()